cypress/integration/navigation.spec.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference types="cypress" />
+
+import links from '../../src/links'
+
+context('The navigation bar', () => {
```

Comment:
What's the longer-term plan to handle navigation? Maybe introducing something like PageObject pattern would be helpful?